packages/asciidoc.rb

```diff
@@ -0,0 +1,24 @@
+require 'package'
+
+class Asciidoc < Package
+  description 'AsciiDoc is a presentable text document format for writing articles, UNIX man pages and other small to medium sized documents.'
+  homepage 'http://asciidoc.org/'
+  version '8.6.8'
+  source_url 'https://downloads.sourceforge.net/project/asciidoc/asciidoc/8.6.8/asciidoc-8.6.8.tar.gz'
+  source_sha256 'ffb67f59dccaf6f15db72fcd04fdf21a2f9b703d31f94fcd0c49a424a9fcfbc4'
+
+  depends_on 'autoconf'
+  depends_on 'python27' unless File.exists? "#{CREW_PREFIX}/bin/python"
+
+  def self.build
+    system "autoconf"
+    system "sed -i 's,/etc/vim,#{CREW_PREFIX}/etc/vim,g' Makefile.in"
```

Comment:
@uberhacker The path is hardcoded.  But Makefile checks `$(DESTDIR)/$(vimdir)` first.  And it wont attempt to install the vim stuff since there is no `/etc/vim` directory.
```
vimdir = ${prefix}/etc/vim
...
install-vim:
        @for d in $(DESTDIR)/$(vimdir) /etc/vim; do \
                if ! test -d $$d; then continue; fi ; \
                echo "installing Vim files in $$d" ; \
                $(INSTALL) -d $$d/syntax ; \
                $(INSTALL_DATA) vim/syntax/asciidoc.vim $$d/syntax ; \
                $(INSTALL) -d $$d/ftdetect ; \
                $(INSTALL_DATA) vim/ftdetect/asciidoc_filetype.vim $$d/ftdetect ; \
        done
```